xrpl/models/transactions/transaction.py

```diff
@@ -91,6 +91,33 @@ def _get_errors(self: Memo) -> Dict[str, str]:
             errors["Memo"] = "Memo must contain at least one field"
         return errors
 
+    @classmethod
+    def from_dict(cls: Type[Memo], value: Dict[str, Any]) -> Memo:
+        """
+        Construct a new Memo from a dictionary of parameters.
+
+        Args:
+            value: The value to construct the Memo from.
+
+        Returns:
+            A new Memo object, constructed using the given parameters.
+
+        Raises:
+            XRPLModelException: If the dictionary provided is invalid.
+        """
+        if "memo" not in value:
```

Comment:
No, that already happens when using `to_xrpl`/`from_xrpl` (it capitalizes stuff). It would actually mess up that functionality to have this be uppercase. See `Signer` for the analogous situation.